doc/release-process.md

```diff
@@ -300,15 +306,15 @@ cat "$VERSION"/*/all.SHA256SUMS.asc > SHA256SUMS.asc
 Both variables are used as a guideline for how much space the user needs on their drive in total, not just strictly for the blockchain.
 Note that all values should be taken from a **fully synced** node and have an overhead of 5-10% added on top of its base value.
 
-To calculate `m_assumed_blockchain_size`:
-- For `mainnet` -> Take the size of the data directory, excluding `/regtest` and `/testnet3` directories.
-- For `testnet` -> Take the size of the `/testnet3` directory.
+To calculate `m_assumed_blockchain_size`, take the size in GiB of these directories:
+- For `mainnet` -> the data directory, excluding the `/regtest`, `/signet`, and `/testnet3` directories and any overly large files, e.g. a huge `debug.log`
+- For `testnet` -> `/testnet3`
```

Comment:
No signet for `m_assumed_blockchain_size`? I mean, I guess it's way too small to be relevant at this point
```
m_assumed_blockchain_size = 1;
m_assumed_chain_state_size = 0;
```
But you do add it for `m_assumed_chain_state_size` below.

Reply:
Good catch! Fixed.